packages/fxa-settings/src/pages/Signin/container.tsx

```diff
@@ -454,27 +406,36 @@ const SigninContainer = ({
             emailVerified, // might not need
           },
         };
-      } catch (error: any) {
+      } catch (error) {
         // If 'invalid token' is received from profile server, it means
         // the session token has expired
-        let { errno } = error;
+        const { errno } = error as AuthUiError;
         if (errno === AuthUiErrors.INVALID_TOKEN.errno) {
-          errno = AuthUiErrors.SESSION_EXPIRED.errno;
           discardSessionToken();
+          return { error: AuthUiErrors.SESSION_EXPIRED };
         }
-        return {
-          data: null,
-          error: {
-            errno,
-            ftlId: composeAuthUiErrorTranslationId(errno),
-            message: AuthUiErrorNos[errno].message,
-          },
-        };
+        return { error };
       }
     },
     [authClient]
   );
 
+  const sendUnblockEmailHandler = useCallback(
+    async (email: string) => {
+      try {
+        await authClient.sendUnblockCode(email);
+        return {};
```

Comment:
Just a nit, for consistency with some other places you could return `success: true` instead of an empty object.